script.js

```diff
@@ -0,0 +1,129 @@
+
+var currentValue='';
+let signPresent=0;
+function appendToDisplay(value) {
+    
+    if((value==="+" || value==="-" || value==="*" || value==="/") && signPresent===0){
+        var display = document.getElementById('display');
+        // console.log(signPresent + "h");
+        // console.log("kk");
+    display.value += value;
```

Comment:
install code formatter, https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode and format your code